src/utils/test-util.ts

```diff
@@ -338,6 +378,7 @@ export async function getOrIssueCredential(
         cred.sad.i === issuerAid.prefix &&
         cred.sad.a.i === recipientAid.prefix &&
         cred.sad.a.AID === credData.AID &&
+        cred.sad.a.engagementContextRole === credData.engagementContextRole &&
```

Comment:
we won't always be getting or issuing ECRs. It could also be OORs. Lets discuss whats in credData and the most flexible way to handle this.